pkg/volume/plugins.go

```diff
@@ -89,6 +89,14 @@ type RecyclableVolumePlugin interface {
 	NewRecycler(spec *Spec) (Recycler, error)
 }
 
+// CreatableVolumePlugin is an extended interface of VolumePlugin and is used to create persistent volumes for the cluster.
+type CreatableVolumePlugin interface {
+	VolumePlugin
+	// NewCreater creates a new volume.Creater which knows how to create this resource
+	// in accordance with the underlying storage provider
+	NewCreater(spec *Spec) (Creater, error)
```

Comment:
Don't want that to block this going in, I just want to call it out.  I don't know if we'll be getting to volume security for shared storage any time soon.